<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LP121" failures="1" tests="7" name="Test" time="120.676" errors="0" timestamp="2024-08-08T18:15:09 IST">
  <testcase classname="appiumautomation.loginPage" name="loginWithoutUsername" time="9.302">
    <skipped/>
  </testcase> <!-- loginWithoutUsername -->
  <testcase classname="appiumautomation.loginPage" name="loginWithoutUsername" time="3.658">
    <skipped/>
  </testcase> <!-- loginWithoutUsername -->
  <testcase classname="appiumautomation.loginPage" name="loginWithoutUsername" time="3.143">
    <failure type="java.lang.AssertionError" message="expected [Username is required1] but found [Username is required]">
      <![CDATA[java.lang.AssertionError: expected [Username is required1] but found [Username is required]
at appiumautomation.loginPage.loginWithoutUsername(loginPage.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- loginWithoutUsername -->
  <testcase classname="appiumautomation.loginPage" name="loginWithoutPassword" time="3.531"/>
  <testcase classname="appiumautomation.loginPage" name="loginWithIncorrectUsernameAndPassword" time="4.252"/>
  <testcase classname="appiumautomation.loginPage" name="loginWithCorrectUsernameAndPassword" time="6.619"/>
  <testcase classname="appiumautomation.EndToEndTest" name="checkoutProduct" time="40.057"/>
</testsuite> <!-- Test -->
